src/daemon/rpc_command_executor.cpp

```diff
@@ -449,7 +449,7 @@ bool t_rpc_command_executor::show_status() {
     % get_sync_percentage(ires)
     % (ires.testnet ? "testnet" : ires.stagenet ? "stagenet" : "mainnet")
     % bootstrap_msg
-    % (!has_mining_info ? "mining info unavailable" : mining_busy ? "syncing" : mres.active ? ( ( mres.is_background_mining_enabled ? "smart " : "" ) + std::string("mining at ") + get_mining_speed(mres.speed) ) : "not mining")
+    % (!has_mining_info ? "mining info unavailable" : mining_busy ? "syncing" : mres.active ? ( ( mres.is_background_mining_enabled ? "smart " : "" ) + std::string("mining at ") + get_mining_speed(mres.speed) + std::string("to ") + mreq.miner_address ) : "not mining")
```

Comment:
I think I made a boo-boo by using `mreq.` instead of `mres.`. I am not sure if `miner_address` is available in `mres.`.

Reply:
Did this compile?
In any case, mres has "address" which is what you want.
Might be worth making this a second line is mining is enabled btw, the address is ~100 characters long already.

Reply:
yeah due to the length, I figured it would only show the first n characters of the address